tests/test_install.sh

```diff
@@ -0,0 +1,157 @@
+#! /bin/sh
+
+test_dir="$(dirname $0)"
+tmp_dir="$(mktemp -d)"
+state_dir="$(mktemp -d)"
+module_dir="${state_dir}/module"
+applycmd_dir="$(mktemp -d)"
+applycmd_log="${tmp_dir}/apply-device-config.log"
+mender_configure="${test_dir}/../src/mender-configure"
+
+# Override variables inside the Update Module.
+export TEST_CONFIG="${state_dir}/device-config.json"
+export TEST_APPLY_CMD="${applycmd_dir}/apply-device-config"
+
+oneTimeTearDown() {
+    rm -rf "${tmp_dir}"
+}
+
+setUp() {
+    mkdir -p "${tmp_dir}"
+    mkdir -p "${state_dir}"
+    mkdir -p "${applycmd_dir}"
+    mkdir -p "${module_dir}"
+
+    cat > "${TEST_APPLY_CMD}" <<EOF
+#!/bin/sh
+
+echo "\$(basename "\$0")" "\$@" >> "${applycmd_log}"
+EOF
+    chmod ugo+x "${TEST_APPLY_CMD}"
+
+    cat > "${module_dir}/meta-data" <<EOF
+{"new":"value"}
+EOF
+
+    echo '{"old":"value"}' > "${TEST_CONFIG}"
+}
+
+tearDown() {
+    rm -rf "${state_dir}" "${applycmd_dir}" "${tmp_dir}"
```

Comment:
As we are recreating `${module_dir}/meta-data` in each `setUp`, we can clean it here too.
```suggestion
    rm -rf "${state_dir}" "${applycmd_dir}" "${tmp_dir} "${module_dir}""
```

Reply:
Yes, but `${module_dir}` is part of `${state_dir}`, so it is cleaned.

Reply:
Right!